lib/matplotlib/tests/test_axes.py

```diff
@@ -3641,16 +3641,16 @@ def test_stem_args():
     # Test the call signatures
     ax.stem(y)
     ax.stem(x, y)
-    ax.stem(x, y, 'r--')
-    ax.stem(x, y, 'r--', basefmt='b--')
+    ax.stem(x, y, linefmt='r--')
+    ax.stem(x, y, linefmt='r--', basefmt='b--')
 
 
 def test_stem_dates():
     fig, ax = plt.subplots(1, 1)
     xs = [dateutil.parser.parse("2013-9-28 11:00:00"),
           dateutil.parser.parse("2013-9-28 12:00:00")]
     ys = [100, 200]
-    ax.stem(xs, ys, "*-")
+    ax.stem(xs, ys, linefmt="*-")
```

Comment:
But the `*` is still ignored here?

Reply:
Sure. Actually, the `linefmt` doesn't do anything at all here. Solid is already the default, so we can just remove the parameter.